src/ngx_http_auth_jwt_module.c

```diff
@@ -293,6 +293,13 @@ static ngx_int_t ngx_http_auth_jwt_handler(ngx_http_request_t *r)
 			jwt_free(jwt);
 		}
 
+		if (!jwtcf->auth_jwt_redirect)
```

Comment:
I'd prefer to swap this and have if be:

```
if (jwtcf->auth_jwt_redirect)
{
    // ... all of the rest of this method ...
    
    return NGX_HTTP_MOVED_TEMPORARILY;
}

return NGX_HTTP_UNAUTHORIZED;
```

Reply:
Am usually the guy to return early, to avoid having more context to worry about ;].

But I reverted it :].